distributor-api-v1/use-cases/how-to-support-payment-cards-in-distributor-client-application.md

```diff
@@ -0,0 +1,19 @@
+# How to support payment cards in Distributor client application
```

Comment:
Maybe we should mention in some way that this guide is only applicable for custom solutions right in the title? People can easily skip the paragraph where this is casually mentioned.

```suggestion
# How to support payment cards in custom Distributor client application
```